src/test/util/setup_common.cpp

```diff
@@ -39,7 +39,8 @@
 #include <scheduler.h>
 #include <script/sigcache.h>
 #include <spork.h>
-#include <statsd_client.h>
+#include <stats/client.h>
+#include <stats/rawsender.h>
```

Comment:
why? doesn't seems as it is needed.
Both `StatsdClient` and `InitStatsClient` are defined in `stats/client.h`

Reply:
```
In file included from test/util/setup_common.cpp:5:
In file included from ./test/util/setup_common.h:9:
In file included from ./chainparamsbase.h:8:
In file included from /usr/bin/../lib/gcc/x86_64-linux-gnu/9/../../../../include/c++/9/memory:80:
/usr/bin/../lib/gcc/x86_64-linux-gnu/9/../../../../include/c++/9/bits/unique_ptr.h:79:16: error: invalid application of 'sizeof' to an incomplete type 'RawSender'
        static_assert(sizeof(_Tp)>0,
                      ^~~~~~~~~~~
/usr/bin/../lib/gcc/x86_64-linux-gnu/9/../../../../include/c++/9/bits/unique_ptr.h:292:4: note: in instantiation of member function 'std::default_delete<RawSender>::operator()' requested here
          get_deleter()(std::move(__ptr));
          ^
./stats/client.h:39:7: note: in instantiation of member function 'std::unique_ptr<RawSender>::~unique_ptr' requested here
class StatsdClient
      ^
/usr/bin/../lib/gcc/x86_64-linux-gnu/9/../../../../include/c++/9/bits/unique_ptr.h:292:4: note: in instantiation of member function 'std::default_delete<StatsdClient>::operator()' requested here
          get_deleter()(std::move(__ptr));
          ^
test/util/setup_common.cpp:186:24: note: in instantiation of member function 'std::unique_ptr<StatsdClient>::~unique_ptr' requested here
    ::g_stats_client = InitStatsClient(*m_node.args);
                       ^
./stats/client.h:17:7: note: forward declaration of 'RawSender'
class RawSender;
      ^
1 error generated.
make[2]: *** [Makefile:13189: test/util/libtest_util_a-setup_common.o] Error 1
```

This is also why `stats/rawsender.h` was included in `stats/client.h`, to avoid the awkwardness of having to include the second header for it to compile as expected.

Reply:
Resolved in latest push!